spec/features/display_peeps_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'pg'
+
+feature 'homepage displays all the peeps' do
+  scenario 'a user is displayed a page with all the peeps' do
+    connection = PG.connect(dbname: 'chitter_test')
+
```

Comment:
Since you have an add-peep route, it may be worth using the feature test to fully test the process of adding the peep then going back to the homepage to view rather than using SQL to fill the database

Reply:
also, the only reason I think of for this test failing is that title will have a nil value, so may be worth adding a title to the inserted data and see how that goes.